lib/va_profile/v3/address_validation/service.rb

```diff
@@ -26,11 +26,10 @@ def address_suggestions(address)
           with_monitoring do
             address.address_pou = address.address_pou == 'RESIDENCE/CHOICE' ? 'RESIDENCE' : address.address_pou
 
+            candidate_res = candidate(address)
```

Comment:
clarifying question: is `candidate_res` referring to "candidate_response" or "candidate_residence"?

Reply:
`        def candidate(address)
          begin
            res = perform(
              :post,
              'candidate',
              address.address_validation_req.to_json
            )
          rescue => e
            handle_error(e)
          end

          res.body
        end`